### ISSUE TITLE
Implement DOM events .on(), .one(), .off()

### ISSUE THREAD

**Issue by ZauberNerd** [garthenweb/jquery-next#29]
http://api.jquery.com/on/
http://api.jquery.com/one/
http://api.jquery.com/off/

**Comment by garthenweb** [comment]
I would propose to call it "once" instead of "one" because it is more declarative. Does anything state against it?

**Comment by ZauberNerd** [comment]
We could implement it as 'once', I too like that name better. But I would suggest to make 'one' an alias to 'once'.

**Comment by garthenweb** [comment]
Why would you like to add an alias? To better fit the jQuery API?

**Comment by ZauberNerd** [comment]
yes.
We could do it like this:

``` javascript
one: function one(...) {
  console.warn('Please use $next.once() instead of one.');
  return this.once(...);
}
```

If this project weren't for the "API consistency" (at least a tiny bit) we could find a lot of other names which were a better fit for the functionality. But until now, we've kept very close to the jQuery API and we even have 'jQuery' in our project name ;)

**Comment by garthenweb** [comment]
Okay, when using warnings it's okay for me!

**Comment by ZauberNerd** [comment]
Closed by #33
